lldb/source/Core/Mangled.cpp

```diff
@@ -56,12 +56,8 @@ static inline bool cstring_is_mangled(llvm::StringRef s) {
 // pointers
 static ThreadSafeDenseMap<const char *, ConstString> *
```

Comment:
This doesn't need to return a pointer. If you feel like doing a little more fixing, there is one caller of this function so it could be easily changed to a reference instead of a pointer.

Reply:
I can have a look next week.

Reply:
thanks